src/hammer-tech/hammer_tech.py

```diff
@@ -30,6 +31,7 @@
 # Access it like hammer_tech.filters.lef_filter
 filters = LibraryFilterHolder()
 
+
```

Comment:
Unnecessary extra line

Reply:
Sorry for that, I didn't check it carefully.